From d6aa76d1f405c7c916f094d3e037d9031a23ca9d Mon Sep 17 00:00:00 2001
From: Jianxu Zheng <jian.xu.zheng@intel.com>
Date: Thu, 18 Sep 2014 19:58:05 +0800
Subject: [PATCH 147/164] atomisp.h: add frame sync compensation interface

add ioctl ATOMISP_IOC_G_DEPTH_SYNC_COMP for frame sync timing compensation.

Change-Id: I9c8033d810c5e8fa075857e84c2d0b47f0eb4fe2
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-8035
Signed-off-by: Jianxu Zheng <jian.xu.zheng@intel.com>
---
 include/linux/atomisp.h |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/atomisp.h b/include/linux/atomisp.h
index 4004315..177ec25 100644
--- a/include/linux/atomisp.h
+++ b/include/linux/atomisp.h
@@ -1205,6 +1205,9 @@ struct atomisp_sensor_ae_bracketing_lut {
 #define ATOMISP_IOC_G_EFFECTIVE_RESOLUTION \
 	_IOR('v', BASE_VIDIOC_PRIVATE + 45, struct atomisp_resolution)
 
+/* for depth mode sensor frame sync compensation */
+#define ATOMISP_IOC_G_DEPTH_SYNC_COMP \
+	_IOR('v', BASE_VIDIOC_PRIVATE + 46, unsigned int)
 
 /*
  * Reserved ioctls. We have customer implementing it internally.
-- 
1.7.9.5

